src/python/grpcio/grpc/_channel.py

```diff
@@ -926,6 +926,11 @@ def __init__(self, target, options, credentials):
         self._call_state = _ChannelCallState(self._channel)
         self._connectivity_state = _ChannelConnectivityState(self._channel)
 
+        # Temporary work around UNAVAILABLE issues
```

Comment:
I think we can close out #11043 after this change.
#9884 is the "Make a proper fix in c-core"